src/pdfkit.cs

```diff
@@ -95,28 +518,48 @@ interface PdfActionResetForm {
 		string [] Fields { get; set; }
 
 		[Export ("fieldsIncludedAreCleared")]
-		bool FieldsIncludedAreCleared  { get; set; }
+		bool FieldsIncludedAreCleared { get; set; }
 	}
 
-	[BaseType (typeof (PdfAction), Name="PDFActionURL")]
+	[iOS (11,0)]
+	[BaseType (typeof (PdfAction), Name = "PDFActionURL")]
 	interface PdfActionUrl {
+
+		[DesignatedInitializer]
 		[Export ("initWithURL:")]
 		IntPtr Constructor (NSUrl url);
 
 		[Export ("URL")]
 		NSUrl Url { get; set; }
 	}
 
-	[BaseType (typeof (NSObject), Name="PDFAnnotation")]
+	[iOS (11,0)]
+	[BaseType (typeof (NSObject), Name = "PDFAnnotation")]
 	interface PdfAnnotation : NSCoding, NSCopying {
+
+		[NoMac] // [Mac (10,13)] Headers do not show changes on Mac
+		[Export ("initWithBounds:forType:withProperties:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (CGRect bounds, NSString annotationType, [NullAllowed] NSDictionary properties);
+
+		[NoMac] // [Mac (10,13)] Headers do not show changes on Mac
+		[Wrap ("this (bounds, annotationType.GetConstant (), properties)")]
+		IntPtr Constructor (CGRect bounds, PdfAnnotationKey annotationType, [NullAllowed] NSDictionary properties);
+
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use '.ctor (CGRect, PDFAnnotationKey, NSDictionary)' instead")]
+		//[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use '.ctor (CGRect, PDFAnnotationKey, NSDictionary)' instead")] // Headers do not show changes on Mac
 		[Export ("initWithBounds:")]
 		IntPtr Constructor (CGRect bounds);
 
 		[Export ("page")]
-		PdfPage Page { get; }
+		PdfPage Page { get; set; }
 
 		[Export ("type")]
-		string Type { get; }
+		string Type { get; set; }
+
+		[NoMac]
+		[Wrap ("PdfAnnotationKeyExtensions.GetValue ((NSString) Type)")]
+		PdfAnnotationKey AnnotationType { get; }
```

Comment:
we should expose the setter too,
ideally hiding the old signature with `#if !XAMCORE_4_0` and putting an `[Obsolete]` on it